nvflare/job_config/fed_job.py

```diff
@@ -104,6 +104,14 @@ def add_external_scripts(self, external_scripts: List):
         for _script in external_scripts:
             self.app.add_ext_script(_script)
 
+    def add_external_dir(self, ext_dir: str):
```

Comment:
Please name the methods consistently in all places, either add_external_dir or add_ext_dir.

Reply:
add_ext_dir() is the internal implementation. add_external_dir() and add_external_scripts() are consistent in the fed_job.py